api/rosetta/configuration.go

```diff
@@ -19,6 +19,9 @@ import (
 	"github.com/optakt/flow-dps/rosetta/meta"
 )
 
+// Configuration represents the configuration parameters of a particular blockchain from
+// the Rosetta API's perspective. It details some blockchain metadata, its supported operations,
+// errors, and more.
```

Comment:
This would be the perfect place to say, `see: <URL to the schema from Rosetta docs>`.

Reply:
I don't think there's a spec for that structure? Isn't it just an internal thing on our side? I guess I can link to https://www.rosetta-api.org/docs/data_api_introduction.html but there's no schema about a configuration structure there 🤔